# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sysvinit/agetty-defaults.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Default to a modern TERM for 99.999% users running this in Xterm and the like.

--- sysvinit-3.04/src/init.c.vanilla	2023-12-16 13:32:43.246059867 +0100
+++ sysvinit-3.04/src/init.c	2023-12-16 13:38:20.303045478 +0100
@@ -1407,7 +1407,7 @@
 					ch->action = RESPAWN;
 					strcpy(ch->id, id);
 					strcpy(ch->rlevel, "2345");
-					sprintf(ch->process, "/sbin/agetty -L -s 115200,38400,9600 %s vt102", dev);
+					sprintf(ch->process, "/sbin/agetty -L -J %s xterm-color", dev);
 					ch->next = NULL;
 					for(old = family; old; old = old->next) {
 						if (strcmp(old->id, ch->id) == 0) {
